### ISSUE TITLE
Launch Crash on 1.5.1

### ISSUE THREAD

**Issue by supersaiyansubtlety** [AterAnimAvis/BlockRenderer#25]
I get a crash fairly quickly after trying to launch with 1.5.1. 

These were generated using just:
- `fabric-api-0.30.0+1.16`
- `block_renderer-1.16.4-forge-fabric-1.5.1`

[crash.txt](https://pastebin.com/U8JNDcWs)

[latest.log](https://pastebin.com/LktK9p9m)

These were generated with the addition of:
- `mixintrace-1.0.0`
- `notenoughcrashes-3.1.4-fabric`

[crash+trace+nec.txt](https://pastebin.com/gW31Nwib)

[log+trace+nec.txt](https://pastebin.com/ggPCjvVP)

**Comment by AterAnimAvis** [comment]
Thanks for the in-depth report / crash logs, whilst it isn't necessary in this case, this would be great normally.

The reason for an early crash was the fact I checked my assuptions after fixing #24 and force an early crash if they're wrong (Rather than allowing for a crash later on in the game from a NullPointerException).

I thought I had checked the fabric side of this (I.E. no crash) but apparently hadn't, it's what I get for pushing whilst feeling sick.

Sorry for this, fix pushed and a new release is on it's way.
